File: src/treesAndGraphs/java/utility/temp.java
package treesAndGraphs.java.utility;

import java.util.Arrays;
import static treesAndGraphs.java.utility.viewBinaryTree.paddingBaseCombo;

public class temp {
	public static void main(String[] args) {
		int n = 50;
		int[] init = new int[n];
		for (int i = 0; i < n; i++) {
			init[i] = i;
		}
		BinaryTreeNode root = BinaryTreeNode.createBinaryTreeNodeFromSortedArray(init);
		viewBinaryTree.printTree(root);

		int[][] combo = paddingBaseCombo(root);
		for(int[] c: combo){
			System.out.println(Arrays.toString(c));
		}
	}
}

File: src/treesAndGraphs/python/utility/temp.py
from treesAndGraphs.python.utility.binary_tree_node import create_binary_tree_from_sorted_array
from treesAndGraphs.python.utility.view_binary_tree import print_tree


def main():
    init = [i for i in range(28)]
    root = create_binary_tree_from_sorted_array(init)
    print_tree(root)


if __name__ == "__main__":
    main()
